state/graphql-sender/pom.xml

```diff
@@ -0,0 +1,116 @@
+    <apollo.version>1.2.2</apollo.version>
+
+    <introspectionFile>${project.build.directory}/graphql-schema/schema.json</introspectionFile>
+    <clientPackage>com.expediagroup.streamplatform.streamregistry.state.graphql</clientPackage>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.expediagroup.streamplatform</groupId>
+      <artifactId>stream-registry-state-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.apollographql.apollo</groupId>
+      <artifactId>apollo-runtime</artifactId>
+      <version>${apollo.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jetbrains</groupId>
+      <artifactId>annotations</artifactId>
+      <version>19.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <scope>provided</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.expediagroup.streamplatform</groupId>
+        <artifactId>stream-registry-graphql-maven-plugin</artifactId>
+        <version>${project.version}</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+            <phase>generate-sources</phase>
+            <configuration>
+              <sourceSdlResource>stream-registry.graphql</sourceSdlResource>
+              <targetIntrospectionFile>${introspectionFile}</targetIntrospectionFile>
+            </configuration>
+          </execution>
+        </executions>
+        <dependencies>
+          <dependency>
+            <groupId>com.expediagroup.streamplatform</groupId>
+            <artifactId>stream-registry-graphql-api</artifactId>
+            <version>${project.version}</version>
+          </dependency>
+        </dependencies>
+      </plugin>
+      <plugin>
+        <groupId>com.github.sparow199</groupId>
+        <artifactId>apollo-client-maven-plugin</artifactId>
+        <version>2.3.2</version>
+        <executions>
+          <execution>
+            <id>graphql-generate-sources</id>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+            <phase>generate-sources</phase>
+            <configuration>
+              <sourceDirName>${project.basedir}/src/main/resources</sourceDirName>
+              <outputDirectory>${project.build.directory}/generated-sources/graphql-client</outputDirectory>
+              <introspectionFile>${introspectionFile}</introspectionFile>
+              <rootPackageName>${clientPackage}</rootPackageName>
+              <customTypeMap>
+                <ObjectNode>com.fasterxml.jackson.databind.node.ObjectNode</ObjectNode>
+              </customTypeMap>
+              <suppressRawTypesWarning>true</suppressRawTypesWarning>
+            </configuration>
+          </execution>
+          <!--          <execution>-->
```

Comment:
Can we remove this if it isn't in use?

Reply:
Yes, I left it there in case I do a full IT.

Reply:
done